src/main/webapp/WEB-INF/user.jsp

```diff
@@ -40,16 +39,6 @@
     <hr/>
     <% }  %>
 
-    <% if(isUserLoggedIn){ %>
```

Comment:
Overall, code looks good - I'm just trying to understand why checks to make sure users are logged in / valid were deleted. If you could clarify that would be great.

Reply:
So I realized when removing the message form on the  user page that we didn't need to check "if isUserLoggedIn and isUserViewingSelf" to show the aboutMe form, because isUserViewing self is initialized in the servlet to be 
boolean isViewingSelf =
        userService.isUserLoggedIn()
            && userData.getEmail().equals(userService.getCurrentUser().getEmail());
Therefore, that check changed to if isViewingSelf because if a user is viewing self, then we show the aboutMe form 
The reason this particular check is removed is because it was for the message form, and for the message form we wanted to view it if the user is logged in - however, we don't have a messageForm anymore, so we no longer need to do that check as we don't show a message form no matter what